config/charts/inferencepool/README.md

```diff
@@ -34,7 +37,6 @@ The following table list the configurable parameters of the chart.
 
 | **Parameter Name**                          | **Description**                                                                                                        |
 |---------------------------------------------|------------------------------------------------------------------------------------------------------------------------|
-| `inferencePool.name`                        | Name for the InferencePool, and endpoint picker deployment and service will be named as `{.Release.name}-epp`.         |
```

Comment:
Why remove the ability to set the InferencePool name?

Reply:
it is redundant, it is already the name of the "release".